From 9531ff92b68aafdb43e8ae1b082df623b316f2db Mon Sep 17 00:00:00 2001
From: syphyr <syphyr@gmail.com>
Date: Wed, 28 May 2025 22:20:14 +0200
Subject: [PATCH] Fix error RTLD_DEFAULT undeclared
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

In file included from ../src/autofit/autofit.c:21:
../src/autofit/ft-hb.c: In function ‘ft_hb_funcs_init’:
../src/autofit/ft-hb.c:99:11: error: ‘RTLD_DEFAULT’ undeclared (first use in this function); did you mean ‘FT_LOAD_DEFAULT’?
     lib = RTLD_DEFAULT;
           ^~~~~~~~~~~~
           FT_LOAD_DEFAULT
---
 src/autofit/ft-hb.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/autofit/ft-hb.c b/src/autofit/ft-hb.c
index 105d14924..6ec9e9f6a 100644
--- a/src/autofit/ft-hb.c
+++ b/src/autofit/ft-hb.c
@@ -16,6 +16,10 @@
  */
 
 
+#ifndef _GNU_SOURCE
+#  define _GNU_SOURCE
+#endif
+
 #include <freetype/freetype.h>
 #include <freetype/internal/ftmemory.h>
 
-- 
2.17.1

